src/components/userGroups/K8sObjectPermissions/K8sPermissons.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import { ReactComponent as AddIcon } from '../../../assets/icons/ic-add.svg'
+import K8sPermissionModal from './K8sPermissionModal'
+import { ReactComponent as Clone } from '../../../assets/icons/ic-copy.svg'
+import { ReactComponent as Delete } from '../../../assets/icons/ic-close.svg'
+import { ReactComponent as Edit } from '../../../assets/icons/ic-pencil.svg'
+import { headerOptions } from './K8sPermissions.utils'
+
+export default function K8sPermissons({ k8sPermission, setK8sPermission }) {
+    const [toggleModal, setToggleModal] = useState<boolean>()
+    const [tempPermission, setTempPermission] = useState()
+    const [selectedPermissionAction, setSelectedPermissionAction] = useState<{
+        action: string
+        index: number
+    }>()
+
+    const editPermission = (permissions, action, index) => {
+        setToggleModal(true)
+        setTempPermission(permissions)
+        setSelectedPermissionAction({
+            action,
+            index,
+        })
+    }
+
+    const creatPermission = () => {
+        setToggleModal(true)
+        setTempPermission(null)
+    }
+
+    const deletePermission = (index) => {
+        const _k8sPermission = [...k8sPermission]
+        _k8sPermission.splice(index, 1)
+        setK8sPermission(_k8sPermission)
+    }
+
+    const closeModal = () => {
+        setToggleModal(false)
+        setSelectedPermissionAction(null)
+    }
+
+    return (
+        <>
+            <div className="anchor pointer flex left mt-16 fs-13 fw-6" onClick={creatPermission}>
+                <AddIcon className="add-svg mr-12" /> Add permission
+            </div>
+            {k8sPermission.length ? (
```

Comment:
Change check to `k8sPermission.length > 0` & use `&&`

Reply:
it show 0 instead of hiding